apps/meteor/app/api/server/v1/voip/rooms.ts

```diff
@@ -218,10 +218,12 @@ API.v1.addRoute(
 			check(this.bodyParams, {
 				rid: String,
 				token: String,
-				comment: Match.Maybe(String),
-				tags: Match.Maybe([String]),
+				options: Match.Maybe({
+					comment: Match.Maybe(String),
+					tags: Match.Maybe([String]),
+				}),
```

Comment:
what is the point of pass an object option instead shallow props?

Reply:
@ggazzo  I went with this after seeing a pattern across our code-base like [here](https://github.com/RocketChat/Rocket.Chat/blob/9578e15300465ff6f1074088b3b35366c567bb89/apps/meteor/app/livechat/server/methods/closeRoom.js#L8) where any params used by EE licence were bundled together. But let me know if you think we should still change this